README.md

```diff
@@ -2,68 +2,100 @@
 
-   > sudo docker run -ti -v /var/run/docker.sock:/var/run/docker.sock -e APP_ID=llm2 -e APP_HOST=0.0.0.0 -e APP_PORT=9080 -e APP_SECRET=12345 -e APP_VERSION=1.0.0 -e NEXTCLOUD_URL='<YOUR_NEXTCLOUD_URL_REACHABLE_FROM_INSIDE_DOCKER>' -e CUDA_VISIBLE_DEVICES=0 -p 9080:9080 --gpus all llm2:latest
+4. Download some example models into the `models` directory:
 
+   ```sh
+   wget -nc -P models https://download.nextcloud.com/server/apps/llm/llama-2-7b-chat-ggml/llama-2-7b-chat.Q4_K_M.gguf
+   wget -nc -P models https://download.nextcloud.com/server/apps/llm/leo-hessianai-13B-chat-bilingual-GGUF/leo-hessianai-13b-chat-bilingual.Q4_K_M.gguf
+   wget -nc -P models https://huggingface.co/Nextcloud-AI/llm_neuralbeagle_14_7b_gguf/resolve/main/neuralbeagle14-7b.Q4_K_M.gguf
+   ```
 
+4. Run the app:
 
-3. **Register the service**
+   ```sh
+   PYTHONUNBUFFERED=1 APP_HOST=0.0.0.0 APP_ID=llm2 APP_PORT=9081 APP_SECRET=12345 APP_VERSION=<APP_VERSION> NEXTCLOUD_URL=http://nextcloud.local python3 lib/main.py
+   ```
 
-   Example assuming you are in the source directory of the cloned repository and the docker image of llm2 was successfully build and is up and running
+5. Register the app with the `manual_install` AppAPI deploy daemon (see AppAPI admin settings in Nextcloud).
 
-    - *Register manually:*
+   With the [Nextcloud Docker dev environment](https://github.com/juliusknorr/nextcloud-docker-dev), you can just run:
 
-      > sudo -u www-data php /var/www/nextcloud/occ app_api:app:unregister llm2 --silent || true
-      sudo -u www-data php /var/www/nextcloud/occ app_api:app:register llm2 manual_install --json-info "{\"appid\":\"llm2\",\"name\":\"Local large language model\",\"daemon_config_name\":\"manual_install\",\"version\":\"1.0.0\",\"secret\":\"12345\",\"host\":\"localhost\",\"port\":9080,\"scopes\":[\"AI_PROVIDERS\", "TASK_PROCESSING"],\"system_app\":0}" --force-scopes --wait-finish
+   ```sh
+   make register
+   ```
 
-## Development installation on bare metal
+   Example if Nextcloud is installed on bare metal instead:
 
-0. [Install Nvidia Drivers an CUDA on your host system](https://gist.github.com/denguir/b21aa66ae7fb1089655dd9de8351a202)
+   ```sh
+   sudo -u www-data php /var/www/nextcloud/occ app_api:app:unregister llm2 --force
+   sudo -u www-data php /var/www/nextcloud/occ app_api:app:register llm2 manual_install --json-info "{\"id\":\"llm2\",\"name\":\"Local large language model\",\"daemon_config_name\":\"manual_install\",\"version\":\"<APP_VERSION>\",\"secret\":\"12345\",\"port\":9081}" --wait-finish
+   ```
 
-1. Create a virtual python environment
+## Development installation using Docker
 
-    > python3 -m venv ./venv && . ./venv/bin/activate
+> [!NOTE]
+> Currently, the Docker image requires that your host system have CUDA/NVIDIA drivers installed and is equipped with a GPU capable of performing the required tasks.
```

Comment:
Is this really the case? 👁️  👁️ 
Or do you mean for *building* the docker image you need CUDA?

Reply:
The Dockerfile uses the CUDA image as the base. I assume CUDA is still needed to run the container, but I can't verify this yet either.